TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/PageTests/Persons/PersonDetail/InductionTests.cs

```diff
@@ -0,0 +1,232 @@
+namespace TeachingRecordSystem.SupportUi.Tests.PageTests.Persons.PersonDetail;
+
+public class InductionTests(HostFixture hostFixture) : TestBase(hostFixture)
+{
+    private static Dictionary<InductionStatus, string> StatusStrings = new() {
+        {InductionStatus.RequiredToComplete, "Required to complete" },
+        {InductionStatus.Exempt, "Exempt" },
+        {InductionStatus.InProgress, "In progress" },
+        {InductionStatus.Passed, "Passed" },
+        {InductionStatus.Failed, "Failed" },
+        {InductionStatus.FailedInWales, "Failed in Wales"}
+    };
+
+    [Fact]
+    public async Task Get_FeatureFlagOff_NoInductionTabShown()
+    {
+        throw new NotImplementedException();
+    }
+
+    [Fact]
+    public async Task Get_WithPersonIdForPersonWithNoQTS_DisplaysExpectedContent()
+    {
+        // Arrange
+        var person = await TestData.CreatePersonAsync();
+        var expectedWarning = "This teacher doesn’t have QTS ";
```

Comment:
Nit:
```suggestion
        var expectedWarning = "This teacher doesn’t have QTS";
```

Reply:
The actual message is much longer - I didn't write the whole thing but maybe I should - possible brittle test vs just testing that the logic delivers the right message and not a different message